components/create-lead/create-lead-form.tsx

```diff
@@ -101,15 +101,15 @@ const CreateLeadForm: React.FC = () => {
         //! Payload object
         const payloadObj = {
           title: formData?.Title,
-          executive_id: 143,
+          // executive_id: 143,
```

Comment:
Why this field `executive_id` is commented?

Reply:
My bad, The Issue fixed